quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -285,7 +289,14 @@ impl Limits {
         max_handshake_duration,
         Duration
     );
-    setter!(with_max_keep_alive_period, max_keep_alive_period, Duration);
+    setter!(
+        /// Sets the period at which the connection will send a ping to its peer.
+        ///
+        /// This setting will not have any effect unless the connection [`keep_alive`] function is
+        /// enabled. Additionally this setting should be smaller than the max_idle_timeout to have any effect.
```

Comment:
Maybe put some context on why one might want to set this value (preventing middleboxes from losing state for UDP flows)